### DISCUSSION TITLE
LD2410

### DISCUSSION THREAD

**Discussion by raptor8999** [arendst/Tasmota#21591] [Support request]
The script handling the LD2410 sensor is incorrectly written. I would really like someone to correct it because it currently turns the presence sensor into a regular motion sensor because it does not send the value 0 to mqtt, i.e. when the person left the room. I don't know how to improve it. Visual Code Studio has been compiling for 5 hours now, I don't know what's going on, I only changed a few minor things.

**Comment by HofeBY** [comment]
VSC often stops working after an automatic update.
I solve this by uninstalling all additional packages (PlattfomIO etc.) in VSC and then VSC.
You don´t have to uninstall your Tasmota source !
After a reboot, I reinstall the current download of VSC and after the first start, I also reinstall PlattformIO.
It is important to ensure that the automatic post-installation routines can run through.
Then it always worked for me with VSC.

Oberfränkische Grüße 

HofeBY

**Reply by raptor8999** [reply]
HofeBy, thank you for your answer. I did it on gitpod. It compiled there without a problem.

**Reply by HofeBY** [reply]
OK, I don´t know, that Gitpod Compiler is VSC.
I use it on Windows 10 with nearly every VSC problems with new version of VSC.
